### PR TITLE
feat: add document documented columns logic

### LINKED ISSUE #99: [bug] asking to document already documented columns isn't running
If all columns are documented I was not able to get a prompt for documenting descriptions which already have a description although there is functionality for it in the code.

```bash
cd tests/
dbt-sugar doc -m my_first_dbt_model --syrup syrup_4 -vv -s public
```

### PR DESCRIPTION
**Pull request by virvirlopez**
# Description
Adding the logic of calling document to already documented columns.

# How was the change tested
Ran it in local.

# Issue
Closes #99

# Checklist

(Ideally, all boxes are checked by the time we merged the PR, if you don't know how to do any of these don't hesitate to say so in the PR and we'll help you out.)

- [X] I formatted my PR name according to [CONTRIBUTING.md](CONTRIBUTING.md)
- [ ] I added a news fragment to help populating the changelog as encouraged in [CONTRIBUTING.md](CONTRIBUTING.md)
- [x] I added "Closes #<issue_number>" in the "Issue Information" section (if no issue, feel free to tick thick the box anyway).

### CODE REVIEW

File: dbt_sugar/core/task/doc.py [hunk 1]
```diff
@@ -191,17 +196,23 @@ def document_columns(self, columns: Dict[str, str]) -> None:
         columns_names = list(columns.keys())
         for i in range(0, len(columns_names), NUMBER_COLUMNS_TO_PRINT_PER_ITERACTION):
```
Comment (bastienboutonnet) [inline dbt_sugar/core/task/doc.py:197]: you know that you can use `ennumerate()` for this kind of thing right?

**Review by bastienboutonnet — DISMISSED** [review]
Looks good but check one of my comments. It's minor but I think you can simplify some of the for loop logic with indexing by using `enumerate()` just in case you want to give that some love.

File: dbt_sugar/core/task/doc.py (hunk 1 above)
Comment (virvirlopez) [reply dbt_sugar/core/task/doc.py:201]: but is not the same, im getting the element by groups with this:
so if NUMBER_COLUMNS_TO_PRINT_PER_ITERACTION = 10
and we have 50 columns
this for will give me 0-9 in the first interaction, then 10-19...

Comment (virvirlopez) [reply dbt_sugar/core/task/doc.py:201]: and I am not interested in the element, only in the index to pass it to the function.

Comment (bastienboutonnet) [reply dbt_sugar/core/task/doc.py:201]: Ah yeah because of pagination!

**Review by bastienboutonnet — APPROVED** [review]
LGTM

**Pull request merged**